cursorless-talon/src/fallback.py

```diff
@@ -0,0 +1,112 @@
+from typing import Callable
+
+from talon import actions
+
+from .versions import COMMAND_VERSION
+
+# This ensures that we remember to update fallback if the response payload changes
+assert COMMAND_VERSION == 7
+
+action_callbacks = {
```

Comment:
Would be cool to support clone; fine leaving out of scope for this PR tho. We should prob file with some other improvements we'd like to make

Reply:
I think I will leave this for a follow up.